### PR TITLE
CLI cheatsheet updated to Humble

### LINKED ISSUE #5: Is the cheat sheet updated (2021)?
Hi there, great work! Is the cheat sheet updated or are there commands that have been deprecated/modified/added?

### PR DESCRIPTION
**Pull request by jkaniuka**
Partially fixes https://github.com/ubuntu-robotics/ros2_cheats_sheet/issues/5. Colcon cheatsheet update remains to be done.

### CODE REVIEW

**Comment by artivis** [comment]
Hi, thanks for the PR :)
I've created a new `humble` branch and changed the PR target so that it is merged in `humble`.
Eventually, `humble` would become the new default branch.

**Comment by jkaniuka** [comment]
Great :smiley: Should I change something in my PR before merging?

**Comment by artivis** [comment]
> Great 😃 Should I change something in my PR before merging?

Nothing to change on your side for now. I'll review this asap and let you know :+1: .

File: cli/cli_cheats_sheet.tex [hunk 1]
```diff
@@ -109,17 +107,22 @@
...
 Examples:
 \\
-\terminal{ros2 info \textangles{bag-name}}\\
-\terminal{ros2 play \textangles{bag-name}}\\
-\terminal{ros2 record -a}
+\terminal{ros2 bag convert \ddash input \textangles{bag-name} storage\_id
+\terminalinebreak~ \ddash output-options \textangles{settings-file-name}}\\
+\terminal{ros2 bag info \textangles{bag-name}}\\
+\terminal{ros2 bag play \textangles{bag-name}}\\
+\terminal{ros2 bag record -a}
```
Comment (artivis) [inline cli/cli_cheats_sheet.tex]: @jkaniuka Could you please add an example of `ros2 bag reindex`?

File: cli/cli_cheats_sheet.tex [hunk 2]
```diff
@@ -109,17 +107,22 @@
 \\
 %
 \begin{tabularx}{\linewidth}{lX}
+\smallhspace \cliverb{convert}      & Write out a new bag with different settings.\\
 \smallhspace \cliverb{info}      & Output information of a bag.\\
+\smallhspace \cliverb{list}      & Output list of available plugins.\\
 \smallhspace \cliverb{play}      & Play a bag.\\
-\smallhspace \cliverb{record}    & Record a bag.
+\smallhspace \cliverb{record}    & Record a bag.\\
+\smallhspace \cliverb{reindex}    & Reconstruct metadata file for a bag.\\
```
Comment (artivis) [inline cli/cli_cheats_sheet.tex]:
```suggestion
\smallhspace \cliverb{reindex}    & Reconstruct metadata file for a bag.
```

File: cli/cli_cheats_sheet.tex [hunk 3]
```diff
@@ -161,8 +164,7 @@
 \hrulefill
 
 %
-\clicmd{doctor} A tool to check ROS setup and other potential issues such as network, package versions,
-rmw middleware etc. \\
+\clicmd{doctor} Check ROS setup and other potential issues \\
```
Comment (artivis) [inline cli/cli_cheats_sheet.tex]:
```suggestion
\clicmd{doctor} Check ROS setup and other potential issues. \\
```

**Review by artivis — CHANGES_REQUESTED** [review]

File: cli/cli_cheats_sheet.tex (hunk 1 above)
Comment (jkaniuka) [reply cli/cli_cheats_sheet.tex]: Added in https://github.com/ubuntu-robotics/ros2_cheats_sheet/pull/6/commits/2cb22ee86c7d261c14f6a32456d84f942ecac103

File: cli/cli_cheats_sheet.tex (hunk 2 above)
Comment (jkaniuka) [reply cli/cli_cheats_sheet.tex]: Fixed in https://github.com/ubuntu-robotics/ros2_cheats_sheet/pull/6/commits/2cb22ee86c7d261c14f6a32456d84f942ecac103

File: cli/cli_cheats_sheet.tex (hunk 3 above)
Comment (jkaniuka) [reply cli/cli_cheats_sheet.tex]: Fixed in https://github.com/ubuntu-robotics/ros2_cheats_sheet/pull/6/commits/2cb22ee86c7d261c14f6a32456d84f942ecac103

**Review by artivis — APPROVED** [review]

**Comment by artivis** [comment]
Thanks for the update :+1: .
Merging.

**Pull request merged**